packages/@lwc/engine-core/src/framework/wiring.ts

```diff
@@ -216,12 +212,18 @@ function createConnector(vm: VM, name: string, wireDef: WireDef): WireAdapter {
                 // Note: when new context arrives, the config will be recomputed and pushed along side the new
                 // context, this is to preserve the identity characteristics, config should not have identity
                 // (ever), while context can have identity
-                computeConfigAndUpdate();
+                if (vm.state === VMState.connected) {
```

Comment:
VMState is set to die soon, is there a way to implement this without using it?

Reply:
maybe some internal flag that u set when connected, and disconnected.

Reply:
@caridy sure, it can be done. there's some other modifications that can also be done to simplify the code, however i will prefer to merge this pr as is (with the least modifications possible to fix the leak), and tackle `vm.state` usage and some refactor of this section in a new PR.

Reply:
hmm, I don't understand the rush here!